src/pages/gallery/index.tsx

```diff
@@ -25,117 +25,140 @@ interface Question {
 }
 
 interface GalleryIndexProps {
-  assessments: Assessment[];
   questions: Question[];
 }
 
-const GalleryIndex: React.FC<GalleryIndexProps> = ({
-  assessments,
-  questions,
-}) => {
+const GalleryIndex: React.FC<GalleryIndexProps> = ({ questions }) => {
   return (
     <React.Fragment>
       <Head>
         <title>Gallery | PrairieLearn</title>
       </Head>
-      <div className="container my-5">
-        <div className="mb-5">
-          <h1 className="display-3">Gallery</h1>
-          <p className="lead">
-            Explore all the functionality PrairieLearn has to offer.
-          </p>
+      <PageBanner
+        title="Gallery"
+        text="Explore all the functionality PrairieLearn has to offer"
+      />
+
+      <div className={classnames("container-fluid py-4 my-5")}>
+        <div className="container-md">
+          <div className="row">
+            <div
+              className="col-md-6 order-2 pt-4
+            "
+            >
+              <Image src={assessmentImage} alt="assessment page view" />
+            </div>
+            <div className="col-md-6 order-1">
+              <HomepageHeading>Assessments</HomepageHeading>
```

Comment:
Nit: we should probably rename this component if we're going to start using it on other pages. Admittedly, this original name of mine was not great 😅